activities/codebase-stewardship/criteria-for-codebase-stewardship.md

```diff
@@ -0,0 +1,20 @@
+---
+type: Resource
+bpmn: lifecycle.bpmn
```

Comment:
I don't feel that this BPMN diagram illustrates the 'criteria' aspect.

Perhaps a `lifecycle.md` file with a short description of what the process is for, and in the future further detail not captured by the BPMN diagram.